src/ripple/core/ClosureCounter.h

```diff
@@ -93,7 +93,7 @@ class ClosureCounter
             ++counter_;
         }
 
-        Wrapper (Wrapper&& rhs)
+        Wrapper (Wrapper&& rhs) noexcept
```

Comment:
This doesn't work because `closure_` is not always nothrow_move_constructible.

Reply:
Good catch. We could make it conditionally noexcept though.

```
Wrapper (Wrapper&& rhs) noexcept(std::is_nothrow_move_constructible<Closure>::value)
```